app/models/application.py

```diff
@@ -44,25 +40,47 @@ def from_dict(cls, d: dict):
             }
         )
 
-    def create_sections(self, application):
+    @staticmethod
+    def create_blank_sections(fund_id, round_id, language):
```

Comment:
Could the config be passed in with a defaulted argument to make this more testable?